app/Domain/Ideas/Templates/advancedBoards.blade.php

```diff
@@ -326,8 +307,7 @@ class="dropdown ticketDropdown userDropdown noBg show right lastDropdown dropRig
                     @endforeach
                 ];
                 leantime.ideasController.initIdeaKanban(ideaStatusList);
-                leantime.canvasController.initUserDropdown('ideas');
-
+                leantime.ideasController.initUserDropdown();
```

Comment:
The function was changed and it needs to be `leantime.canvasController.initUserDropdown('ideas');`